### DISCUSSION TITLE
Error found using memtest_vulkan Standard 5-minute test - 3070ti

### DISCUSSION THREAD

**Discussion by Hardolihno** [GpuZelenograd/memtest_vulkan#40] [Card-specific memtest_vulkan error logs]
1: Bus=0x07:00 DevId=0x2482   8GB NVIDIA GeForce RTX 3070 Ti
Standard 5-minute test of 1: Bus=0x07:00 DevId=0x2482   8GB NVIDIA GeForce RTX 3070 Ti
      1 iteration. Passed  0.0995 seconds  written:    3.2GB 118.5GB/sec        checked:    6.5GB  90.2GB/sec
     11 iteration. Passed  1.0532 seconds  written:   32.5GB 111.5GB/sec        checked:   65.0GB  85.3GB/sec
     69 iteration. Passed  5.0311 seconds  written:  188.5GB 139.2GB/sec        checked:  377.0GB 102.5GB/sec
    450 iteration. Passed 30.0398 seconds  written: 1238.2GB 154.1GB/sec        checked: 2476.5GB 112.6GB/sec
    880 iteration. Passed 30.0090 seconds  written: 1397.5GB 173.3GB/sec        checked: 2795.0GB 127.4GB/sec
Error found. Mode INITIAL_READ, total errors 0x1 out of 0x34000000 (0.00000011%)
Errors address range: 0x18FFE8644..=0x18FFE8647  iteration:1316
values range: 0x27507D12..=0x27507D12   FFFFFFFF-like count:0    bit-level stats table:
         0x0 0x1  0x2 0x3| 0x4 0x5  0x6 0x7| 0x8 0x9  0xA 0xB| 0xC 0xD  0xE 0xF
SinglIdx                 |                 |   1             |
TogglCnt       1         |                 |                 |
1sInValu                 |                 |                 |            1

   1477 iteration. Passed 30.0329 seconds  written: 1940.2GB 230.4GB/sec        checked: 3880.5GB 179.6GB/sec
   2081 iteration. Passed 30.0313 seconds  written: 1963.0GB 233.6GB/sec        checked: 3926.0GB 181.5GB/sec
   2689 iteration. Passed 30.0203 seconds  written: 1976.0GB 235.5GB/sec        checked: 3952.0GB 182.7GB/sec
   3293 iteration. Passed 30.0069 seconds  written: 1963.0GB 233.8GB/sec        checked: 3926.0GB 181.7GB/sec
   3898 iteration. Passed 30.0319 seconds  written: 1966.2GB 234.1GB/sec        checked: 3932.5GB 181.8GB/sec
   4506 iteration. Passed 30.0072 seconds  written: 1976.0GB 235.3GB/sec        checked: 3952.0GB 182.9GB/sec
   5114 iteration. Passed 30.0085 seconds  written: 1976.0GB 235.5GB/sec        checked: 3952.0GB 182.8GB/sec
Standard 5-minute test fail - ERRORS FOUND

**Comment by Hardolihno** [comment]
I've ran the test again, but had set my soc vcore back to auto and it did not have any errors at all. I'll run a 1 hour OCCT VRAM test and see if I get errors again (Previous 100% VRAM test gave 5 errors within 30 minutes).

**Reply by galkinvv** [reply]
The overall memory GB/sec bandwidth usage in test seems too low for 3070Ti, the normal speed is much faster. And testing on lower speeds tend to have less effective memory errors reveal.

Do you have any other apps running in the background?

Or maybe your setup has huge initial memory usage, try creating .bat file to run with a smaller memory usage as suggested here:
https://github.com/GpuZelenograd/memtest_vulkan/issues/38#issuecomment-2029426674
